upcloud.php

```diff
@@ -167,7 +167,7 @@ private function getRowRules(&$vars)
                 'valid_connection' => [
                     'rule' => [
                         [$this, 'validateConnection'],
-                        $vars['api_token'],
+                        $vars['api_base_url'],
```

Comment:
This looks a bit odd on surface, but the previously passed `$vars['api_token']` was actually a superfluous leftover from 69e3fccee6789a542b928278a8bb536e73cd67ba.